packages/salesforcedx-apex-replay-debugger/src/core/heapDumpService.ts

```diff
@@ -320,4 +377,19 @@ export class HeapDumpService {
   ) {
     return varContainer.name === extentValueEntry.keyDisplayValue;
   }
+
```

Comment:
The Trigger context variables are Boolean, List and Map types and will have symbols lists of variables that starts with "Trigger.".